pages/blog/blog-post-template.html

```diff
@@ -31,8 +31,13 @@ <h2>Blog Post Title Written by the Author</h2>
         </section>
 
         <article class="blog_post_article">
-            <img src="https://www.pngkey.com/png/detail/233-2332677_image-500580-placeholder-transparent.png" alt="A generic body image and alt text">
+
+            <div class="blog_post_article--image">
+                <img src="https://www.pngkey.com/png/detail/233-2332677_image-500580-placeholder-transparent.png" alt="A generic body image and alt text">
+            </div>
+
```

Comment:
I've gone ahead and wrapped up our image into a CSS class so that we can add width/height easily and keep it responsive for all devices😄